src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -59,7 +74,7 @@ public double getArmPosition() {
         return -((abs > ArmConstants.ARM_LIMIT && abs < 1) ? (deg - 360) : deg);
     }
 
-    public double calculateFeedforward(double positionRadians, double velocity) {
+    public double calculateFeedforward(double positionRadians) {
         return feedforward.calculate(positionRadians, 0);
     }
```

Comment:
May remove as feedforward is unused (since setArmProfiled will replace setArmPosition)